ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -3,6 +3,12 @@ all:
     CLEAN_UNIT_TESTS:
       hosts:
         cvp-instance-ips-cvaas
+    BACKBONE_UNIT_TESTS:
```

Comment:
I have seen issues with molecule, after adding the `CLEAN_UNIT_TESTS`since they have a different `fabric_name` and the documentation can only render for one fabric.
Maybe we can just change to `fabric_name: 'all'` everywhere under unit_tests.

Reply:
Changed to all for my tests.